plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -662,7 +679,8 @@ def main():
     boot_opts = {
         'bootdevice': module.params['bootdevice'],
         'uefi_target': module.params['uefi_target'],
-        'boot_next': module.params['boot_next']
+        'boot_next': module.params['boot_next'],
+        'boot_override_mode': module.params['boot_override_mode']
```

Comment:
I'd suggest to add a trailing comma here so the next addition has a smaller diff:
```suggestion
        'boot_override_mode': module.params['boot_override_mode'],
```